toml_test.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import "testing"
+
+func TestTOMLFile(t *testing.T) {
+	configFile := "test/config.toml"
+
+	// Uncomment the code below if you want to generate the test/config.toml file
```

Comment:
Generating the test file isn't necessary. This commented section can be removed.